frontend/integration-tests/tests/cluster-settings.scenario.ts

```diff
@@ -95,6 +95,7 @@ describe('Cluster Settings', () => {
     expect(crudView.actionForLabel('Explore Console API').isDisplayed()).toBe(true);
     await crudView.actionForLabel('Explore Console API').click();
 
+    await clusterSettingsView.globalConfigDetailsTitleIsLoaded;
     expect(clusterSettingsView.globalConfigDetailsTitle.isDisplayed()).toBe(true);
```

Comment:
This check isn't needed if we use `browser.wait` above.

Reply:
Which `browser.wait` are you refering too?

Reply:
Are you saying I don't need the `await ...`?

Reply:
No, I'm saying you don't need the `expect` on line 99. You've already confirmed the title is there on the line above.

Reply:
ah, ok, removed.